dotnet/src/Connectors/Connectors.Memory.Postgres/Database.cs

```diff
@@ -67,20 +67,33 @@ public async Task CreatePgVectorExtensionAsync(NpgsqlConnection conn, Cancellati
     /// <returns></returns>
     public async Task CreateTableAsync(NpgsqlConnection conn, int vectorSize, CancellationToken cancellationToken = default)
     {
-        await this.CreatePgVectorExtensionAsync(conn, cancellationToken).ConfigureAwait(false);
-
         using NpgsqlCommand cmd = conn.CreateCommand();
 #pragma warning disable CA2100 // Review SQL queries for security vulnerabilities
         cmd.CommandText = $@"
             CREATE TABLE IF NOT EXISTS {TableName} (
                 collection TEXT,
                 key TEXT,
-                metadata TEXT,
+                metadata JSONB,
                 embedding vector({vectorSize}),
                 timestamp BIGINT,
                 PRIMARY KEY(collection, key))";
 #pragma warning restore CA2100 // Review SQL queries for security vulnerabilities
         await cmd.ExecuteNonQueryAsync(cancellationToken).ConfigureAwait(false);
+
+        await this.MigrateAsync(conn, cancellationToken).ConfigureAwait(false);
```

Comment:
What is the plan to discontinue the column type migration operation?

Reply:
I plan to remove the migration operation after the next two releases.